index.js

```diff
@@ -1,9 +1,16 @@
 'use-strict';
 
-var xml = require('xml');
-var Base = require('mocha').reporters.Base;
-var filePath = process.env.MOCHA_FILE || 'test-results.xml';
-var fs = require('fs');
+var fs      = require('fs');
+var path    = require('path');
+var mkdirp  = require('mkdirp');
+
+var xml     = require('xml');
+var Base    = require('mocha').reporters.Base;
+
+function FILE_PATH() {
```

Comment:
@Pharb made a change similar to this in PR #9. 

The problem is that when you change MOCHA_FILE between runs/reports the new path isn't picked up. (This could also be alleviated by @Pharb's PR making the path configurable in reporterOptions)